cdap-app-fabric/src/main/java/co/cask/cdap/gateway/handlers/SecureStoreHandler.java

```diff
@@ -86,7 +89,7 @@ public void delete(HttpRequest httpRequest, HttpResponder httpResponder, @PathPa
   public void get(HttpRequest httpRequest, HttpResponder httpResponder, @PathParam("namespace-id") String namespace,
                   @PathParam("key-name") String name) throws Exception {
     SecureKeyId secureKeyId = new SecureKeyId(namespace, name);
-    String data = new String(secureStoreService.get(secureKeyId).get(), StandardCharsets.UTF_8);
+    String data = new String(secureStore.getSecureData(namespace, name).get(), StandardCharsets.UTF_8);
```

Comment:
Strictly speaking, you don't need to convert the `byte[]` to `String` and call the `sendString` (which in turn will turn the `String` back to `byte[]`). You can simply do `responder.sendContent(OK, ChannelBuffers.wrappedBuffer(data), "text/plain; charset=utf-8", null);`